File: python/high-scores/high_scores.py
def latest(scores):
    return scores[-1]

def personal_best(scores):
    return max(scores)

def personal_top_three(scores):
    #scores_list= scores
    #scores_list.sort(reverse = True)
    #print(sorted(scores, reverse= True)[:3])
    return sorted(scores, reverse= True)[:3]

personal_top_three([1,13])

File: csharp/high-scores/HighScores.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class HighScores
{
    private List<int> _scores;
    public HighScores(List<int> list)
    {
        _scores = new List<int>(list);
    }

    public List<int> Scores()
    {
        return _scores;
    }

    public int Latest()
    {
        return _scores[_scores.Count -1];
    }

    public int PersonalBest()
    {
        var bestScore = int.MinValue;
        foreach (int score in _scores)
        {
            bestScore = Math.Max(bestScore, score);
        }
        return bestScore;
    }

    public List<int> PersonalTopThree()
    {
        List<int> sorted_scores = _scores.OrderByDescending(c => c).ToList();
        if (_scores.Count < 3)
        {
            return sorted_scores;
        } else 
        {
        return sorted_scores.GetRange(0,3);
        }
    }
}